scio-core/src/main/scala/com/spotify/scio/experimental/package.scala

```diff
@@ -164,20 +176,33 @@ package object experimental {
      *
      * // Read from [myproject:samples.gsod] instead.
      * bq.getTypedRows[Row]("myproject:samples.gsod")
+     *
+     * // Read from a query instead.
+     * sc.getTypedRows[Row]("SELECT * FROM [publicdata:samples.gsod] LIMIT 1000")
```

Comment:
Everything looks 👍 the only thing is that the example should really be a table if it does `SELECT * ...`, I know it's tiny, but even BQ doc says in most cases you should not do `SELECT * ...`